src/index.js

```diff
@@ -74,10 +77,14 @@ export default function ({ types: t, traverse: _traverse }) {
         const { parentPath } = path;
         const explicitReturn = parentPath.isReturnStatement();
         const implicitReturn = parentPath.isArrowFunctionExpression();
-
-        const openingElement = elementOpenCall(path.get("openingElement"), this);
-        const closingElement = elementCloseCall(path.get("openingElement"), this);
-        const children = buildChildren(path.get("children"), this);
+        const openingElementPath = path.get("openingElement");
+
+        const openingElement = elementOpenCall(openingElementPath, this);
+        const closingElement = elementCloseCall(openingElementPath, this);
+        const isSkipping = hasSkip(openingElementPath.get("attributes"), this) ||
+          isComponent(openingElementPath.get("name"), this);
```

Comment:
/cc @mairatma, as you have [custom component](https://github.com/jridgewell/babel-plugin-transform-incremental-dom/pull/43) code to handle this case.